vllm/engine/arg_utils.py

```diff
@@ -208,6 +209,14 @@ def add_cli_args(
                             help='maximum context length covered by CUDA '
                             'graphs. When a sequence has context length '
                             'larger than this, we fall back to eager mode.')
+        parser.add_argument(
+            '--kv-cache-dtype',
+            type=str,
+            choices=['auto', 'fp8_e5m2'],
+            default='auto',
+            help=
+            'Data type for kv cache storage. If "auto", will use model data type. Not FP8 is not supported when cuda version is lower than 11.8.'
```

Comment:
Please make sure every line is fewer than 80 characters.

Reply:
done